app/shared/api-services/poll/poll-factory.tests.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+describe('Answer Factory', function () {
+  var $httpBackend,
+  data,
+  pollFactory,
+  pollId;
+
+  beforeEach(module('apiServices.poll.factory'));
+
+  beforeEach(inject(function(_$httpBackend_, _Poll_) {
+    pollFactory = _Poll_;
+    $httpBackend = _$httpBackend_;
+
+  }));
+
+  describe('getPoll()', function () {
+    it('makes a get request', function () {
+      pollId = 27;
+      pollFactory.getPoll(pollId);
+      $httpBackend.expectGET('/cms/api/v1/poll/27').respond(201);
```

Comment:
Similar to what @spra85 mentioned above, check that the code can handles the response payload in an expected way.